projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/checkout/asm-cost-center.e2e-spec.ts

```diff
@@ -55,19 +65,23 @@ context('B2B - ASM Account Checkout', () => {
 
     cy.wait('@costCenterReq').its('response.statusCode').should('eq', 400);
     alerts.getErrorAlert().contains('Invalid cost center.');
-  });
 
-  it('should not show error on valid cost center', () => {
-    alerts.getErrorAlert().then(() => {
-      alerts.getErrorAlert().within(() => {
+    cy.log('--> should not show error on valid cost cente');
+    alerts.getErrorAlert().then((alert) => {
+      cy.wrap(alert).within(() => {
         cy.get('button').click();
       });
     });
-    cy.intercept('PUT', '*costcenter?costCenterId=*').as('costCenterReq');
+    // flaky:  added timer between intercepters
+    cy.wait(1000);
```

Comment:
The same request is intercepted twice, causing errors. A solution possibly would be to remove this and line 77, though it may cause issues.